### PR TITLE
add plot util

### PR DESCRIPTION
**Pull request by ritch**
Adds a utility to `foo.types.Object()` for quickly defining a plot object property.

## Summary by CodeRabbit

- **New Features**
  - Added enhanced functionality to the dropdown component to support schema-based operations.
  - Introduced a new plotting method in the operators to facilitate advanced data visualization.

- **Bug Fixes**
  - Improved data handling in the plotting component to ensure stability and correctness.

- **Chores**
  - Implemented a new constant to manage panel loading timeouts more effectively.

### CODE REVIEW

File: app/packages/core/src/plugins/SchemaIO/components/DropdownView.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export default function DropdownView(props) {
             Array.isArray(value) && type !== "array"
               ? value.join(separator)
               : value;
-          onChange(path, computedValue);
+          onChange(path, computedValue, schema);
```
Comment (ritch) [inline app/packages/core/src/plugins/SchemaIO/components/DropdownView.tsx:79]: Also adds a missing schema value here. This is needed for triggering change events for enum properties.

File: fiftyone/operators/types.py [hunk 2]
```diff
@@ -275,6 +275,19 @@ def grid(self, name, **kwargs):
         self.define_property(name, obj, view=grid)
         return obj
 
+    def plot(self, name, **kwargs):
+        """Defines a plot view as a :class:`View`.
```
Comment (ritch) [inline fiftyone/operators/types.py:279]: This isn't quite right....

Comment (ritch) [reply fiftyone/operators/types.py:279]: Should be...

```
Defines an object property displayed as a plot.
```

**Review by imanjra — APPROVED** [review]
LGTM

**Pull request merged**